css/ampshared.css

```
@@ -132,6 +133,22 @@
   right: 0;
 }
 
+/* In previous rule and in this one, make sure noscript fallbacks get fill layout as if they are placeholders */
+body noscript > * {
+  position: absolute !important;
+
+  /* Override width/height attributes defined on HTML elements in noscript. */
+  width: 100%;
+  height: 100%;
+
+  /* Supersede sibling [placeholder]/[fallback] z-index:1 */
+  z-index: 2;
+}
+
+noscript {
```

Technically this should only be targeting `noscript` elements that are children of AMP elements, but at the very least it should be restricted to the `body`:

```suggestion
body noscript {
```

---

This issue was specificity. I'm happy with whatever targets `noscript` directly, instead of `not(noscript)`.